<?xml version="1.0"?>
<launch>
    <node pkg="mavros_extras" type="copter_visualization" name="copter_visualization">
        <param name="fixed_frame_id" value="odom"/>
        <param name="child_frame_id" value="base_link"/>
        <param name="num_rotors" value="4"/>
    </node>
    <node name="rviz" type="rviz" pkg="rviz" args="-d $(find foresight)/config/rviz.rviz"/>
</launch>
